mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -255,9 +255,42 @@ def bind(self, bus):
             self.event_scheduler.set_id(self.skill_id)
             self._enclosure = EnclosureAPI(bus, self.name)
             self._register_system_event_handlers()
+
             # Initialize the SkillGui
             self.gui.setup_default_handlers()
 
+            # TODO can we rename this namespace to mycroft.playback.XXX ?
```

Comment:
small note, this renaming seems to make even more sense now according to SPEC https://docs.google.com/document/d/1ebS60QLna4_AOQxQYpoaZrpkBzPXjL-7VFPJ7-xVvIQ/edit#heading=h.pr2syrcbyvq6

but out of scope for this PR

Reply:
Yeah agreed. 

I do however wonder if these handlers should also live in mycroft.enclosure.gui?

Reply:
self.gui is initialized before the bus is ready, the events can only be registered when/after the skill bind method is called

since the SkillGUI class takes the skill object as a param (which means it is not used outside skills) i don't see anything wrong in having those here, moving them around would require some refactoring to ensure event handlers were properly registered and increase complexity

Reply:
disregard previous comment, just noticed that SkillGUI has a setup_default_handlers method called in the skill bind method

it makes more sense to move these to the gui class since they dont use the skill class for anything, good catch!